src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -95,6 +121,32 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
             ])
+            ->add('show_catalog_price_rules', IconButtonType::class, [
+                'icon' => 'visibility',
+                'label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
+                'attr' => [
+                    'data-hide-label' => $this->trans('Hide catalog price rules', 'Admin.Catalog.Feature'),
+                    'data-show-label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
+                    'data-show-icon' => 'visibility',
+                    'data-hide-icon' => 'visibility_off',
+                ],
+            ])
+            ->add('catalog_price_rules', CatalogPriceRulesType::class, [
+                'label' => $this->trans('Catalog price rules', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'attr' => [
+                    'data-catalog-price-url' => $catalogPriceRuleEditLink,
+                ],
+                'external_link' => [
+                    'text' => $this->trans('[1]Manage catalog price rules[/1]', 'Admin.Catalog.Feature'),
+                    'href' => $catalogPriceRuleIndexLink,
+                    'align' => 'left',
+                ],
+                'row_attr' => [
+                    'id' => 'catalog-price-rules-container',
+                    'class' => 'd-none',
+                ],
+            ])
```

Comment:
I think all those options could be moved into the `CatalogPriceRulesType` since its its responsibility, this way it makes this form type simpler and the responsibilities are correctly separated

Reply:
Moved everything into CatalogPriceRulesType. Now it's a bit odd there because PriceSummaryType has everything inside the type, while SpecificPricesType has all settings in PricingType, but I am guessing because it only has label things.